### ISSUE TITLE
'neo' is not available yet

### ISSUE THREAD

**Issue by jarmol** [neohaskell/NeoHaskell#114]
Tried to install neo using the given curl instruction

Result:  curl --proto '=https' --tlsv1.2 -sSf https://sh.neohaskell.org | sh
curl: (6) Could not resolve host: sh.neohaskell.org

**Comment by NickSeagull** [comment]
Hello! Thanks for the interest in NeoHaskell 😄 

The docs are currently a placeholder, as [per this note on the docs](https://neohaskell.org/docs/docs-intro/). You are not the first person encountering this issue, so I'm sorry if this made stuff confusing for you.

I'm gonna prioritize removing the placeholder docs to avoid confusing future users.

You're very welcome to join the Discord server to discuss and say what you like/dislike from programming so we can build the best language ever 😊

**Comment by jarmol** [comment]
Thanks for reply
Place holder means, neo is not yet available?
OK, I'll wait untill there is a live link to download neo.
Meanwhile I continue using Haskell 9.4.8

**Comment by NickSeagull** [comment]
> Place holder means, neo is not yet available?

Yes, that is correct. I will change the title of this issue to track the availability of neo and to come back to you once is done. Thanks for understanding!

**Comment by NickSeagull** [comment]
This is fixed now, one can follow the instructions [in the documentation](https://neohaskell.org/docs/getting-started/installing-neohaskell/) to install the `neo` CLI tool.
